opencensus/proto/export/export.proto

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "io.opencensus.export";
+option java_outer_classname = "ExportProto";
+
+option go_package = "github.com/census-instrumentation/opencensus-proto/gen-go/exportproto";
+
+package export;
+
+import "trace/trace.proto";
+import "stats/metrics/metrics.proto";
+
+service ExportService {
+  rpc Export (ExportRequest) returns (ExportResponse) {}
+}
+
+message ExportRequest {
+  repeated opencensus.proto.trace.Span spans = 1;
```

Comment:
I think for consistency we have two options:
1) add a SpanSet similar to MetricSet
2) use repeated Metric

I would prefer 1).

Reply:
I went with the option 2 because it is not quite clear when we will use SpanSet elsewhere at this moment. PTAL.

Reply:
Why use inline batching here instead of using a streaming RPC? This seems like an ideal use for streaming RPCs because you could avoid having to allocate a large batch of Span objects and you also don't have to manually batch.

Reply:
The method takes repeated Spans because we wanted to give the clients the ability to batch before putting things on wire. Otherwise, we can avoid having a repeated fields.

Reply:
we may want to rationalize this with https://github.com/envoyproxy/data-plane-api/blob/1dcdaaaf0853c5258aa9d1588a052ca51dbbfa7d/envoy/service/trace/v2/trace_service.proto

I like their approach of using both a repeated field and streaming RPC - that way the client gets maximum freedom to optimize it's batching and memory usage. With a streaming request, I'm not sure there is any reason to do batching of objects inside the application rather than just sending them as soon as they are available and allowing the batching to occur lower down the stack.